QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151102_002922	$Psychokino Tome Volume 1
+QUEST_20151102_002923	$Story of the first Psychokino
+QUEST_20151102_002924	$Psychokino Tome Volume 2
+QUEST_20151102_002925	$Story of Psychokino's magic
+QUEST_20151102_002926	$Psychokino Tome Volume 3
+QUEST_20151102_002927	$Story of Psychokinos before the Medzio Diena
+QUEST_20151102_002928	$Psychokino Tome Volume 4
+QUEST_20151102_002929	$What Psychokino did at the Mage Tower
+QUEST_20151102_002930	$Psychokino Tome Volume 5
+QUEST_20151102_002931	$Psychokino's Potential
+QUEST_20151102_002932	After the lord's migration has fallen, the people of the south are heading towards the Orsha.{nl}If you're an adventurer who has recently started your travel, head to camping area of the east.{nl}There should be people who's able to guide you.
+QUEST_20151102_002933	I saw Orsha from a slope on the way here from a town.{nl}The splendor surrounded by castle wall I've seen as a kid has disappeared and now coiled with trees with the size of a house.{nl}I wonder how long you can last in such a place.
+QUEST_20151102_002934	It's obviously intelligence for wizard and its affiliation.{nl}However, you also need to remember spirit to cast spells frequently.{nl}Select well between balance and concentration.
+QUEST_20151102_002935	It's obviously strength for archer and its affiliation.{nl}However, you cannot forget nimble dexterity.{nl}It's best to think ahead of what to become.
+QUEST_20151102_002936	It's obviously intelliegence for cleric and its affiliation.{nl}However, you may also require strength or spirit.{nl}It's best to think ahead of what to become.
+QUEST_20151102_002937	Only if our town also had an inn for merchants to stay and rest.{nl}We could've prepared beforehand..{nl}After spending so long at an inlet of a valley, we got to suffer for not knowing the world affairs.
+QUEST_20151102_002938	Living there inside of Orsha's partially destroyed fortress.{nl}Living here with swarming monsters and fear of safety.{nl}Both are tough choices to make.{nl}I wish to return back to my hometown..
+QUEST_20151102_002939	Please let the people with the smell of the ocean to head down to Dallanas.{nl}It's starting to become uneasy with the people not to be seen as time passes.{nl}By any chance, did they all leave to Orsha without me?
+QUEST_20151102_002940	There's no way that such a huge amount of people can enter Orsha.{nl}That's why we try to gather and accommodate them here as much as possible.
+QUEST_20151102_002941	Lethargic and fear can be seen in people's eyes.{nl)What would you do if you were the village chief?
+QUEST_20151102_002942	{nl}It's the elderly's complaint..{nl}I worry for the lord..
+QUEST_20151102_002943	Those who are usually lazy are still carefree in these time of danger.{nl}People who were suppose to arrive earlier has yet to come.
+QUEST_20151102_002944	The land has been protected by the ancestors of the family.{nl}I will protect the land, no matter the trouble.{nl}Please lend me your power.
```

Comment:
"This land" (2x) seems more appropriate. 

no matter the trouble = no matter what hardship comes my way?